theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/Dot.tsx

```diff
@@ -59,10 +59,10 @@ const HitZone = styled.div`
     pointer-events: auto;
     &:hover:after {
       position: absolute;
-      top: calc(50% - ${snapCursorSize / 2}px);
-      left: calc(50% - ${snapCursorSize / 2}px);
-      width: ${snapCursorSize}px;
-      height: ${snapCursorSize}px;
+      top: calc(50% - ${SNAP_CURSOR_SIZE / 2}px);
+      left: calc(50% - ${SNAP_CURSOR_SIZE / 2}px);
+      width: ${SNAP_CURSOR_SIZE}px;
+      height: ${SNAP_CURSOR_SIZE}px;
```

Comment:
Nit: Instead of doing a `/ 2` every time, perhaps have a `SNAP_CURSOR_SIZE_HALF` value.

Random and totally just trivia: in microbenchmarks, floating point division is a magnitude more expensive for the CPU than multiplication.

Reply:
Interesting trivia. I will elect not to modify this because the division is only being performed once when loading.